### PR TITLE
Add failed login scenario

### LINKED ISSUE #41: Handle failed login scenario

### PR DESCRIPTION
**Pull request by shanear**
fixes #41

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -4,7 +4,11 @@ def new
   end
 
   def create
-    return render "new" unless current_user.authenticate(authentication_params)
+    unless current_user.authenticate(authentication_params)
+      flash[:error] = "Login ID and password did not work. Please try again."
```
Comment (orischwartz) [inline app/controllers/sessions_controller.rb]: I know this is the message we used in CF-classic...but "did not work" feels awkward to me. Maybe something like 

> The email and password you entered don't match, please try again.

**Comment by orischwartz** [comment]
:+1: LGTM, waiting on Travis

File: app/controllers/sessions_controller.rb (hunk 1 above)
Comment (shanear) [inline app/controllers/sessions_controller.rb]: @KHarshawat thoughts?

**Comment by shanear** [comment]
@orischwartz :turtle:

File: app/controllers/sessions_controller.rb (hunk 1 above)
Comment (orischwartz) [inline app/controllers/sessions_controller.rb]: BTW, that's the same phrasing both Google and Yahoo use on failed logins:

> The email and password you entered don't match.

**Pull request merged**